kernel-headers-2.6.25-0.40.rc1.git2.fc9.x86_64

In file included from /usr/include/asm/ptrace.h:4,
                 from ../../gdb/amd64-linux-nat.c:44:
/usr/include/asm/ptrace-abi.h:92: error: expected specifier-qualifier-list before ‘u32’

--- gdb-6.7.1-13.fc9.x86_64/gdb/amd64-linux-nat.c-orig	2008-02-15 15:37:28.000000000 -0500
+++ gdb-6.7.1-13.fc9.x86_64/gdb/amd64-linux-nat.c	2008-02-15 15:40:13.000000000 -0500
@@ -35,13 +35,13 @@
 #include <sys/procfs.h>
 #include <asm/prctl.h>
 /* FIXME ezannoni-2003-07-09: we need <sys/reg.h> to be included after
-   <asm/ptrace.h> because the latter redefines FS and GS for no apparent
+   <sys/ptrace.h> because the latter redefines FS and GS for no apparent
    reason, and those definitions don't match the ones that libpthread_db
    uses, which come from <sys/reg.h>.  */
 /* ezannoni-2003-07-09: I think this is fixed. The extraneous defs have
    been removed from ptrace.h in the kernel.  However, better safe than
    sorry.  */
-#include <asm/ptrace.h>
+#include <sys/ptrace.h>
 #include <sys/reg.h>
 #include "gdb_proc_service.h"
 
